test failure:
```
======================================================================
FAIL: test_split_column_missing (couchexport.tests.test_schema.SavedSchemaTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 113, in test_split_column_missing
    self._test_split_column([[None, None, None, None, None]])
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 89, in _test_split_column
    self.assertIn(tuple(row), tuple_rows)
AssertionError: (None, None, None, None, None) not found in set([(None, None, None, None, u'---')])
```

--- a/corehq/ex-submodules/couchexport/models.py
+++ b/corehq/ex-submodules/couchexport/models.py
@@ -391,7 +391,7 @@ class ExportTable(DocumentSchema):
                 return i
 
     def get_items_in_order(self, row):
-        from couchexport.export import SCALAR_NEVER_WAS
+        from couchexport.export import scalar_never_was
         row_data = list(row.get_data())
         for column in self.columns:
             # If, for example, column.index references a question in a form
@@ -404,7 +404,7 @@ class ExportTable(DocumentSchema):
                 i = self.row_positions_by_index[column.index]
                 val = row_data[i]
             except KeyError:
-                val = SCALAR_NEVER_WAS
+                val = scalar_never_was
 
             if issubclass(type(column), ComplexExportColumn):
                 for value in column.get_data(val):

--- a/corehq/ex-submodules/couchexport/tests/test_schema.py
+++ b/corehq/ex-submodules/couchexport/tests/test_schema.py
@@ -110,11 +110,11 @@ class SavedSchemaTest(TestCase):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
+        self._test_split_column([[SCALAR_NEVER_WAS] * 5])
 
     def test_split_column_missing_ignore_extras(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None]], ignore_extras=True)
+        self._test_split_column([[SCALAR_NEVER_WAS] * 4], ignore_extras=True)
 
     def test_split_column_None_ignore_extras(self):
         self.post_it(multi=None)

--- a/corehq/ex-submodules/couchexport/tests/test_schema.py
+++ b/corehq/ex-submodules/couchexport/tests/test_schema.py
@@ -110,7 +110,7 @@ class SavedSchemaTest(TestCase):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, None]])
+        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
 
     def test_split_column_missing_ignore_extras(self):
         self.post_it()

